.github/workflows/tests.yml

```diff
@@ -0,0 +1,34 @@
+name: Tests
```

Comment:
You need to have one GHA actions for it to start in PRs. I'm not sure if there is another way.